SQLClient/src/cmd/root.go

```diff
@@ -0,0 +1,37 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"log"
+	"os"
+)
+
+const rootStr = "SQLClient"
+
+var Verbose bool
+
+var RootCMD = &cobra.Command{
+	Use:   rootStr,
+	Short: fmt.Sprintf("%s is a command line interface (CLI) to interact with the Elections database", rootStr),
+	Run: func(cmd *cobra.Command, args []string) {
+		//qm := queryMaker.NewQueryMaker()
```

Comment:
yeah we can do that too (although it should just run anyways if you malform any args (or dont pass in any) which is this exact same use case

ie.
if I just called go `run ./main.go` with no args it would show help anyways already